src/__testutils__/noop.ts

```diff
@@ -0,0 +1,3 @@
+export function noop() {
```

Comment:
`export default function noop()` ? There is only one export and it has the same name os the file.

Reply:
I used to follow this: https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html and using named exports is just an old habit for me.